src/test/FuzzerImpl.cpp

```diff
@@ -542,6 +555,21 @@ TransactionFuzzer::initialize()
             }
         }
 
+        if (!ops.empty())
+        {
+            auto txFramePtr = createFuzzTransactionFrame(
```

Comment:
Identical code repetition, please refactor into a function or lambda. Probably a function, because you can use the same code in the future when you do result checking for trust lines, etc.

Reply:
Good point.  Done.